studio/schemas/documents/imageWithText.js

```diff
@@ -19,5 +19,10 @@ export default {
             title: 'Image',
             type: 'mainImage',
         },
+        {
+            name: 'id',
+            title: 'Id',
+            type: 'string',
+        },
```

Comment:
What is this field for?

Reply:
anchor links, the logic is on the FE and i forgot to add it in sanity

Reply:
So how would this work? A user would have to input the correct string into the sanity field? Or are you planning to add these programmatically?

Reply:
@asasmith I guess not, I can hardcode the anchor link instead